### PR TITLE
chore(dead code): remove unused invocation_finished on scheduler TaskCreationResult

### PR DESCRIPTION
**Pull request by seriousben**
## Context

 
 CreationResult has an invocation_finished attribute that is misleading since it is not used even though we have lots of logic around it in the scheduler.

## What

Delete the attribute and simplify code setting it.

## Contribution Checklist

- [x] If the python-sdk was changed, please run `make fmt` in `python-sdk/`.
- [x] If the server was changed, please run `make fmt` in `server/`.
- [x] Make sure all PR Checks are passing.

### CODE REVIEW

File: server/task_scheduler/src/lib.rs [hunk 1]
```diff
@@ -15,22 +15,15 @@ pub struct TaskCreationResult {
...
-    pub invocation_finished: bool,
     pub invocation_id: String,
 }
 
 impl TaskCreationResult {
-    pub fn no_tasks(
-        namespace: &str,
-        compute_graph: &str,
-        invocation_id: &str,
-        invocation_finished: bool,
```
Comment (diptanu) [inline server/task_scheduler/src/lib.rs:27]: Why was this removed? There were cases an invocation was still running but we were not creating new tasks.

File: server/task_scheduler/src/task_creator.rs [hunk 2]
```diff
@@ -30,7 +30,6 @@ pub async fn handle_invoke_compute_graph(
             compute_graph: event.compute_graph.clone(),
             new_reduction_tasks: vec![],
             processed_reduction_tasks: vec![],
-            invocation_finished: false,
```
Comment (diptanu) [inline server/task_scheduler/src/task_creator.rs:33]: This should be invocation_finished: true no?

Comment (seriousben) [reply server/task_scheduler/src/task_creator.rs:33]: That is what is going to happen, but that attribute was not read anywhere.

In Create tasks, it checks for outstanding tasks to finalize.
